sidecred.go

```diff
@@ -20,6 +20,14 @@ type CredentialRequest struct {
 	// name is up to the individual provider.
 	Name string `json:"name"`
 
+	// Rotation is an override for the default rotation window
+	// measured in seconds.
+	// This will aid in cases where we want to be more granular
+	// for possibly longer running authentications or processes.
+	//
+	// Needs to be a int because JSON unmarshalling does not support Duration
```

Comment:
This comment would end up in the package documentation so I think its best to just remove it :)

Reply:
:+1: